test/e2e/traceflow_test.go

```diff
@@ -53,7 +53,6 @@ type testcase struct {
 // TestTraceflow is the top-level test which contains all subtests for
 // Traceflow related test cases so they can share setup, teardown.
 func TestTraceflow(t *testing.T) {
-	skipIfHasWindowsNodes(t)
```

Comment:
IntraNode tests run on the first Node which is the Linux controlplane Node.
InterNode tests run on the first and the second Node, which is still a Linux Node?

Reply:
Thanks. Corrected the NodeName and PodImage. Re-run the tests.